pkg/runtime/embedded.go

```diff
@@ -24,9 +24,9 @@ import (
 )
 
 type encodable struct {
-	e        Encoder `json:"-"`
+	E        Encoder `json:"-"`
 	obj      Object
-	versions []unversioned.GroupVersion `json:"-"`
+	Versions []unversioned.GroupVersion `json:"-"`
```

Comment:
This was "Line 29: error: struct field versions has json tag but is not exported (vet)"

The better fix is to remove the quotes around the - ?